developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_28.rst

```diff
@@ -51,6 +51,7 @@ Due to the popularity of CLI tools for development of Nextcloud apps, the likeli
 Added APIs
 ^^^^^^^^^^
 
+* ``\OCP\DB\Events\AddMissingIndicesEvent`` to add missing indices to the database schema.
```

Comment:
Should go into a "Added events" section similar to the ones I'm adding for "Removed events" in https://github.com/nextcloud/documentation/pull/10737/files

Reply:
Done.